### ISSUE TITLE
Popup window is empty half the time

### ISSUE THREAD

**Issue by EpocDotFr** [rugk/awesome-emoji-picker#74]
## Bug description

Since a very recent update of the extension (I'm not able to tell you the exact version, sorry), the popup window showed after clicking the action button (in the top bar) is empty half the time.

## Steps to reproduce

It's reproducible at least in version 1.2 of the extension.

1. Install the addon
2. Click on the action button
3. The popup window may or may not be empty
4. Click anywhere on the Firefox window so the popup window is closed (may it be empty or not)
5. Click on the action button again
6. The popup window may or may not be empty depending on what happened in step 3

### Actual behavior

The popup window is empty half the time when clicking on the action button.

![image](https://user-images.githubusercontent.com/2053431/72537636-c1fe7380-387c-11ea-9431-82d3047a06b1.png)

### Expected behavior

The popup window show emojis every time I click on the action button.

## System

Operating system and version: Windows 10
Browser and version: Firefox 72.0.1 64 bits
Add-on version: 1.2
Other addons: Bitwarden, uBlock Origin

## Possible solution

No errors in the browser console.

**Comment by rugk** [comment]
Thanks for reporting this, so some questions:
* How often does this happen? Really in 50% of the cases, or more/less; or maybe only once after start?
* If you've opened it again (after it is empty) the issue does _not_ happen again, I guess?

If so, could you help by debugging this a little? I would provide the steps, if you need.

**Comment by EpocDotFr** [comment]
> How often does this happen? Really in 50% of the cases, or more/less; or maybe only once after start?

It's really happening 50% of the time. In other words: when I click on the action button, exactly 1 time out of 2 the popup is empty.

> If you've opened it again (after it is empty) the issue does not happen again, I guess?

I've recorded a GIF demonstrating the issue:

![b](https://user-images.githubusercontent.com/2053431/72600058-2cfa8980-3913-11ea-96f9-6ae0d3b62437.gif)

**Comment by rugk** [comment]
That is really strange.

So here is how to debug a popup in short steps:
1. [Open a new tab and go to `about:debugging` -> "This Firefox"](https://extensionworkshop.com/documentation/develop/debugging/#developer-tools-toolbox).
2. Find this add-on there and click "Debug".
3. In the menu, [disable that the popup is hidden when you click it away](https://extensionworkshop.com/documentation/develop/debugging/#debugging-popups), so you can inspect the inside.
4. If you have done so, you can open the extension popup and close it (by pressing <kbd>Esc</kbd> as clicking anywhere else does not work now, because of the previous option).
  Do not forget to select the "popup" inside the debugger, to debug the correct part of the extension. This is only explained at the bottom in the docs:
   > If your extension has multiple HTML documents open, click the page select icon (pages selector) to open the popup’s document.

Thanks in advantage for helping.

If you are there, could you please check whether there any errors/console messages in the console, when the issue occurs?
You could also enable to not clear the log when the popup opens ("a new website opens") to see all the log messages at once. Just make sure to indicate which part is from a broken display and which is from a working one.

**Comment by EpocDotFr** [comment]
Oops, forgot to mention it's happening on a PC at my workplace, which is having Firefox always running 5 days a week. I started my workstation this morning, the issue is gone. So I guess this issue can be disregarded 😅

**Comment by rugk** [comment]
Sad to see, cause it would have been an interesting test.
Anyway, if it happens again, please just notify me/open a new issue or so.
